foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
+
+%description
+The package contains binaries of Grub and Grub2 bootloaders for PXE/BOOTP use
+on TFTP Foreman Proxy extracted from Red Hat Enterprise Linux 6 and 7. This
+noarch package contains binaries for multiple architectures, including x86_64,
+i386, ARM64, PPC64 and PPC64 LE.
+
+
+%prep
+%setup -q -n tftpboot
+
+
+%build
+
+
+%install
+install -d -m0755 %{buildroot}%{_sharedstatedir}/tftpboot
+cp -p -r grub grub2 %{buildroot}%{_sharedstatedir}/tftpboot
```

Comment:
This is from Foreman installation:

```
[root@hp-dl2x170g6-01 ~]# ll /var/lib/tftpboot -d
drwxr-xr-x. 6 root root 135 May 29 12:38 /var/lib/tftpboot
[root@hp-dl2x170g6-01 ~]# ll /var/lib/tftpboot
total 136
drwxr-xr-x. 2 foreman-proxy root   166 May 29 12:10 boot
-rw-r--r--. 1 root          root 20832 May 29 12:38 chain.c32
-rw-r--r--. 1 root          root 26268 May 29 12:38 memdisk
-rw-r--r--. 1 root          root 55140 May 29 12:38 menu.c32
drwxr-xr-x. 2 foreman-proxy root     6 May 29 12:38 poap.cfg
-rw-r--r--. 1 root          root 26826 May 29 12:38 pxelinux.0
drwxr-xr-x. 2 foreman-proxy root   189 May 29 12:56 pxelinux.cfg
drwxr-xr-x. 2 foreman-proxy root     6 May 29 12:38 ztp.cfg
```

This is from Foreman 1.15 actually:

```
[root@zzzap ~]# ll /var/lib/tftpboot -d
drwxr-xr-x. 8 root root 177 30. kvě 16.09 /var/lib/tftpboot
[root@zzzap ~]# ll /var/lib/tftpboot
celkem 172
drwxr-xr-x. 2 foreman-proxy root     6 30. kvě 16.09 boot
drwxr-xr-x. 2 foreman-proxy root     6 30. kvě 16.09 grub
drwxr-xr-x. 2 foreman-proxy root    57 30. kvě 16.10 grub2
-rw-r--r--. 1 root          root 20704 30. kvě 16.03 chain.c32
-rw-r--r--. 1 root          root 33628 30. kvě 16.03 mboot.c32
-rw-r--r--. 1 root          root 26140 30. kvě 16.03 memdisk
-rw-r--r--. 1 root          root 55012 30. kvě 16.03 menu.c32
drwxr-xr-x. 2 foreman-proxy root     6 30. kvě 16.09 poap.cfg
drwxr-xr-x. 2 foreman-proxy root     6 30. kvě 16.09 pxelinux.cfg
-rw-r--r--. 1 root          root 26764 30. kvě 16.03 pxelinux.0
drwxr-xr-x. 2 foreman-proxy root     6 30. kvě 16.09 ztp.cfg
```

You are right, I need to make sure that both grub and grub2 dirs are owned by foreman-proxy with 7xx perms, I will explicitly change that. New dependency will be required there.